src/lib/web-feed-info.js

```diff
@@ -0,0 +1,33 @@
+import { database_pool } from './database.js';
+
+class feedInfo {
+  static async getErrors(publisherId, feedId) {
+
+    const client = await database_pool.connect();
+
+    const feedErrorsSqlQuery = `
+     SELECT
+        raw_data.id as "csDataId",
+        raw_data.validation_results
+      FROM raw_data
+      LEFT JOIN publisher_feed ON raw_data.publisher_feed_id=publisher_feed.id
```

Comment:
Technically this doesn't need a LEFT JOIN and a  publisher_feed.publisher_id=$1 , just  raw_data.publisher_feed_id=$2 would do. But I'm guessing this is to check the passed feed ID actually belongs to this publisher? (That could be a separate simple SQL if needed). No JOIN would probably be more efficient but that's not a blocker, just to note for later if efficiency comes up. (tho if it does paging would also be a good shout)

Also I think it can just be a JOIN, LEFT not needed - there should be no nulls on that condition. But don't think that matters so not a blocker.

Reply:
Good thought, yeah, I wanted to do it in one query to avoid multiple trips to the DB as I know that's quite slow on our setup.